.github/workflows/ci.yml

```diff
@@ -58,3 +58,25 @@ jobs:
           # TODO(gcmn) Enable all tests except notap once we can run them on the CI
           bazel query '//... except attr("tags", "notap", //...) except //bindings/... except //integrations/... except //iree/hal/vulkan:dynamic_symbols_test except //iree/samples/rt:bytecode_module_api_test' | xargs bazel test --test_output=errors
 
+  cmake:
+    runs-on: ubuntu-18.04
+    env:
+      CXX: clang++
+      CC: clang
+      PYTHON_BIN: /usr/bin/python3
+    steps:
+      - name: Installing dependencies
+        run: |
+          sudo apt-get install clang libsdl2-dev python3 python3-pip
+          sudo pip install numpy
+      - name: Checking out latest version and all submodules
+        run: |
+          git config --global remote.origin.fetch '+refs/pull/*:refs/remotes/origin/pull/*'
+          git clone https://github.com/$GITHUB_REPOSITORY $GITHUB_WORKSPACE --no-checkout --no-tags
+          git checkout $GITHUB_SHA
+          git submodule update --init --depth 1000 --jobs 8
+      - name: Building and testing with cmake
+        run: |
+          mkdir build && cd build
+          cmake -DIREE_BUILD_SAMPLES=OFF ..
```

Comment:
We could make the options explicit here, rather than rely on the default values:

```
option(IREE_BUILD_COMPILER "Builds the IREE compiler." OFF)
option(IREE_BUILD_TESTS "Builds IREE unit tests." ON)
option(IREE_BUILD_SAMPLES "Builds IREE sample projects." ON)
option(IREE_BUILD_DEBUGGER "Builds the IREE debugger app." OFF)
```

i.e. 
`cmake -DIREE_BUILD_COMPILER=OFF -DIREE_BUILD_TESTS=ON -DIREE_BUILD_SAMPLES=OFF -DIREE_BUILD_DEBUGGER=OFF`

The defaults that the project sets for developers and the list of targets that we want to build here could be different (devs may just want the basic runtime, CI should test everything that it can)

Reply:
Good point, done.